catalog/spatial/csw/spatial-csw-source/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/source/TestCswSource.java

```diff
@@ -209,9 +210,16 @@ public void testPropertyIsLikeQuery()
         }
         GetRecordsType getRecordsType = captor.getValue();
 
-        String xml = getGetRecordsTypeAsXml(getRecordsType, CswConstants.VERSION_2_0_2);
-        LOGGER.debug(xml);
-        assertXMLEqual(xml, getRecordsControlXml202);
+        String xml = getGetRecordsTypeAsXml(getRecordsType);
```

Comment:
Looks like this block of code is copied multiple times.  Any chance of extracting a method?